packages/api/src/web3Middleware/index.ts

```diff
@@ -111,21 +109,16 @@ export class Web3Middleware {
     this.intervals.push(interval)
   }
 
-  async readContractsState ({ feedContract, proxyContract }: Contracts) {
+  async readContractsState ({ feedContract }: Contracts) {
     try {
-      const feedContractState = {
-        lastPrice: await feedContract.methods.lastPrice().call(),
-        lastTimestamp: await feedContract.methods.timestamp().call(),
-        lastRequestId: await feedContract.methods.lastRequestId().call()
-      }
-      const drTxHash = await proxyContract.methods
-        .readDrTxHash(feedContractState.lastRequestId)
-        .call()
       return {
-        ...feedContractState,
-        drTxHash: toHex(drTxHash).slice(2)
+        lastPrice: await feedContract.methods.lastPrice().call(),
+        // lastResponse contains { timestamp, drTxHash }
+        lastResponse: await feedContract.methods.lastResponse().call(),
+        requestId: await feedContract.methods.requestId().call()
       }
     } catch (err) {
+      console.log('readContractsState ERROR', err)
```

Comment:
✨ I AM THE `console.log` GUARDIAN. ALWAYS HAPPY TO DISTURB YOU WITH MY ANNOYANCE ✨
```suggestion
      console.error('readContractsState ERROR', err)
```